card/client.go

```diff
@@ -74,8 +74,8 @@ func (c *Client) CaptureChargeWithContext(ctx context.Context, data *CaptureChar
 }
 
 // GetCharge gets a card charge
-func (c *Client) GetCharge(data *CaptureChargeParams) (*xendit.CardCharge, *xendit.Error) {
```

Comment:
Isn't this going to be a breaking change?

Reply:
Can we keep it to be a minor release?
I don't see any needs for major releae

Reply:
I think this was a bug from previously

Reply:
So are we going to release it as a major version then?

Reply:
Hmm I think just keep it in this minor. I think we haven't gotten complaints cos no one is using this feature yet